### PR TITLE
Sprint 1/issue 3/create event list screen

### PR DESCRIPTION
**Pull request by AnaCarrizales**
"Iniciar" button now shows the headers of the table "Lista de eventos"

Screenshot from 2024-01-25 19-06-13

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,9 +1,16 @@
 Rails.application.routes.draw do
-  resources :planeadores_de_eventos
-  root 'home#index'
+
+  #resources: es un método utilizado para declarar rutas RESTful (get,post,put,delete,update,etc)}
```
Comment (juperameza) [inline config/routes.rb]: Don't leave comments like this in your project

File: test/controllers/event_controller_test.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 require "test_helper"
 
-class PlaneadoresDeEventoTest < ActiveSupport::TestCase
+class EventControllerTest < ActionDispatch::IntegrationTest
   # test "the truth" do
   #   assert true
   # end
```
Comment (juperameza) [inline test/controllers/event_controller_test.rb]: If you don't need it delete it

File: test/models/event_test.rb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 require "test_helper"
 
-class EventPlannerTest < ActiveSupport::TestCase
+class EventTest < ActiveSupport::TestCase
   # test "the truth" do
   #   assert true
   # end
```
Comment (juperameza) [inline test/models/event_test.rb]: If you don't need this delete it

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -5,14 +5,11 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
-
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
```
Comment (juperameza) [inline app/views/layouts/application.html.erb:8]: This isn't the best way to add bootstrap to  rails
Try to change it in another pr

**Review by juperameza — CHANGES_REQUESTED** [review]

File: app/views/home/_header.html.erb [hunk 5]
```diff
@@ -15,7 +15,7 @@
     <div class="collapse navbar-collapse" id="navbarText">
       <ul class="navbar-nav mr-auto">
         <li class="nav-item active">
-          <%= link_to "Iniciar", { controller: "planeadores_de_eventos" },
+          <%= link_to "Iniciar", { controller: "event" },
```
Comment (MauricioNC) [inline app/views/home/_header.html.erb]: Please pass the route prefix as path.

**Review by MauricioNC — CHANGES_REQUESTED** [review]

**Review by MauricioNC — APPROVED** [review]
LGTM

**Review by juperameza — APPROVED** [review]
LGTM

**Pull request merged**